harness/compareArray.js

```diff
@@ -27,7 +27,7 @@ compareArray.isSameValue = function(a, b) {
 };
 
 compareArray.format = function(spreadable) {
-  return `[${[...spreadable].map(String).join(', ')}]`;
+  return `[${[].map.call(spreadable, String).join(', ')}]`;
```

Comment:
Dunno if this will work--a "spreadable" is an iterable, right?

Reply:
there is a whole problematic over this compareArray file and this format is being only used for the assertion message.

The assertion requires the values a and b (actual and expected) to be accessed in a for loop during the assertion, and the values would be accessed again for this formatting purposes. 

I believe this `[].map.call` is a conservative approach and I'd stick with it for discussing compareArray in a different PR.